shared/middleware/helpers/table.rb

```diff
@@ -79,3 +79,161 @@ def to_a()
   end
 
 end
+
+require 'aws-sdk'
+
+#
+# DynamoTable
+#
+class DynamoTable
+  
+  class NotFound < Sinatra::NotFound
+  end
+
+  def initialize(channel_id, storage_id, table_name)
+    channel_owner, @channel_id = storage_decrypt_channel_id(channel_id) # TODO(if/when needed): Ensure this is a registered channel?
```

Comment:
@geoffrey-elliott @wjordan  what are your thoughts on whether to check if this is a registered channel? This would cause us to hit the database on every table access, right?